### PR TITLE
Add AUTO_PICKUP_SMALL_ITEMS option.

### LINKED ISSUE #19603: "Auto pickup 0 vol light items" does nothing since "all items have volume" update
This can be found in the options menu under the general tab. Can we please have an updated version of this? If it can be reworked to simply accept both volume and weight as a variable, it would be a simple fix.

### PR DESCRIPTION
**Pull request by kevingranade**
Fixes #19603 by adding a new option to select maximum volume to automatically pickup.

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -813,8 +813,13 @@ void options_manager::init()
         false
         );
 
-    add("AUTO_PICKUP_ZERO", "general", _("Auto pickup 0 vol light items"),
-        _("Auto pickup items with 0 Volume, and weight less than or equal to [option] * 50 grams. '0' disables this option"),
+    add("AUTO_PICKUP_ZERO", "general", _("Auto pickup light items"),
```
Comment (Coolthulhu) [inline src/options.cpp]: Could use a name change.

**Comment by kevingranade** [comment]
Do we have migration support for changing option names?

**Comment by Coolthulhu** [comment]
I don't think we do, but I'm pretty sure we don't have migration support for changing option types either.

File: src/options.cpp [hunk 2]
```diff
@@ -813,8 +813,13 @@ void options_manager::init()
         false
         );
 
-    add("AUTO_PICKUP_ZERO", "general", _("Auto pickup 0 vol light items"),
-        _("Auto pickup items with 0 Volume, and weight less than or equal to [option] * 50 grams. '0' disables this option"),
+    add("AUTO_PICKUP_WEIGHT_LIMIT", "general", _("Auto pickup light items"),
+        _("Auto pickup items with weight less than or equal to [option] * 50 grams. You must also set the small items option.  '0' disables this option"),
```
Comment (BrettDong) [inline src/options.cpp]: Minor: one space between the first and the second sentence, but two spaces between the second and the third. And there is no period in the last sentence.

**Pull request merged**